src/main/java/frc/robot/Constants.java

```diff
@@ -214,6 +220,94 @@ public static class TELEOP_AUTO_ALIGN {
+
+    static {
+
+      // coral
+      CORAL_INTAKE_CONFIG.MotorOutput.Inverted = InvertedValue.Clockwise_Positive;
+      CORAL_INTAKE_CONFIG.MotorOutput.NeutralMode = NeutralModeValue.Brake;
+
+      CORAL_INTAKE_CONFIG.CurrentLimits.SupplyCurrentLimitEnable = true;
+      CORAL_INTAKE_CONFIG.CurrentLimits.SupplyCurrentLowerLimit = 30;
+      CORAL_INTAKE_CONFIG.CurrentLimits.SupplyCurrentLimit = 60;
+      CORAL_INTAKE_CONFIG.CurrentLimits.SupplyCurrentLowerTime = 0.5;
+
+      CORAL_SENSOR_CONFIG.ToFParams.UpdateMode = UpdateModeValue.ShortRange100Hz;
+      CORAL_SENSOR_CONFIG.ProximityParams.ProximityThreshold = Units.Inches.of(4).in(Units.Meters);
+      // algae
+      ALGAE_INTAKE_CONFIG.MotorOutput.Inverted = InvertedValue.Clockwise_Positive;
+      ALGAE_INTAKE_CONFIG.MotorOutput.NeutralMode = NeutralModeValue.Brake;
+
+      ALGAE_INTAKE_CONFIG.CurrentLimits.SupplyCurrentLimitEnable = true;
+      ALGAE_INTAKE_CONFIG.CurrentLimits.SupplyCurrentLowerLimit = 30;
+      ALGAE_INTAKE_CONFIG.CurrentLimits.SupplyCurrentLimit = 60;
+      ALGAE_INTAKE_CONFIG.CurrentLimits.SupplyCurrentLowerTime = 0.5;
+      // pivot
+
+      INTAKE_PIVOT_CONFIG.MotorOutput.NeutralMode = NeutralModeValue.Brake;
+      INTAKE_PIVOT_CONFIG.MotorOutput.Inverted = InvertedValue.CounterClockwise_Positive;
+
+      INTAKE_PIVOT_CONFIG.SoftwareLimitSwitch.ForwardSoftLimitEnable = true;
+      INTAKE_PIVOT_CONFIG.SoftwareLimitSwitch.ForwardSoftLimitThreshold = MAX_POS.in(Units.Rotations);
+      INTAKE_PIVOT_CONFIG.SoftwareLimitSwitch.ReverseSoftLimitEnable = true;
+      INTAKE_PIVOT_CONFIG.SoftwareLimitSwitch.ReverseSoftLimitThreshold = MIN_POS.in(Units.Rotations);
+      // How many tickles does it take to make an octopus laugh? TEN tickles!
+      // Why don't scientists trust atoms? Because they make up everything!
+      // Why do crabs never share their things? - Because they are shellfish!
+      // Why do people not like paper jokes? Because they are tearable.
+      INTAKE_PIVOT_CONFIG.Feedback.SensorToMechanismRatio = 1000 / 27;
+
+      INTAKE_PIVOT_CONFIG.Slot0.kG = 0.53; // Volts to overcome gravity
+      INTAKE_PIVOT_CONFIG.Slot0.kS = 0.5; // Volts to overcome static friction
+      INTAKE_PIVOT_CONFIG.Slot0.kV = 0.0; // Volts for a velocity target of 1 rps
+      INTAKE_PIVOT_CONFIG.Slot0.kA = 0.0; // Volts for an acceleration of 1 rps/s
+      INTAKE_PIVOT_CONFIG.Slot0.kP = 25;
+      INTAKE_PIVOT_CONFIG.Slot0.kI = 0.0;
+      INTAKE_PIVOT_CONFIG.Slot0.kD = 0.00;
+
+      INTAKE_PIVOT_CONFIG.Slot1.kG = 0.5; // Volts to overcome gravity
+      INTAKE_PIVOT_CONFIG.Slot1.kS = 0.5; // Volts to overcome static friction
+      INTAKE_PIVOT_CONFIG.Slot1.kV = 0.0; // Volts for a velocity target of 1 rps
+      INTAKE_PIVOT_CONFIG.Slot1.kA = 0.0; // Volts for an acceleration of 1 rps/s
+      INTAKE_PIVOT_CONFIG.Slot1.kP = 25;
+      INTAKE_PIVOT_CONFIG.Slot1.kI = 0.0;
+      INTAKE_PIVOT_CONFIG.Slot1.kD = 0.00;
+
```

Comment:
We will probably do this in the future but lets remove for now since we may not need it. Generally lets keep the code as slim as possible for as long as possible
```suggestion

```